src/services/doppler-api-client.ts

```diff
@@ -61,6 +91,25 @@ export class HttpDopplerApiClient implements DopplerApiClient {
     };
   }
 
+  private mapSubscriberFields(data: any): Fields[] {
+    return data.map((x: any) => ({
+      name: x.name,
+      value: x.value,
+      predefined: x.predefined,
+      private: x.private,
+      readonly: x.readonly,
+      type: x.type,
```

Comment:
It seems to be redundant

Reply:
I am seeing that in other places there happen the same.

I understand that it could help to decouple the model from API, maybe you can map only used fields?

Reply:
yes, when i build the component i can refactor with the fields we need